pom.xml

```diff
@@ -67,7 +68,7 @@
    </dependency>
      <dependency>
       <groupId>org.apache.hadoop</groupId>
-      <artifactId>hadoop-client</artifactId>
```

Comment:
Can you elaborate on why this change is needed? hadoop-client should be the default entry point for consumers of the APIs